templates/myid.j2

```diff
@@ -1 +1,9 @@
+{% if install_mode == "standalone" %}
 {{ zoo_id }}
+{% elif install_mode == "cluster" %}
+{% for server in play_hosts %}
```

Comment:
After experimenting with the same approach to assign the id using the host order, I'm okay using it. Still not comfortable using `play_host` the way it is though.